routes/index.js

```diff
@@ -1,9 +1,51 @@
-var express = require('express');
-var router = express.Router();
+const express = require('express');
 
-/* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+const router = express.Router();
+
+const { google } = require('googleapis');
+const keys = require('../gkeys.json');
+
+/* GET data from Google Sheets API */
+router.get('/umma-server/getData', (req, res) => {
+  const client = new google.auth.JWT(
+    keys.client_email,
+    null,
+    keys.private_key,
+    ['https://www.googleapis.com/auth/spreadsheets.readonly']
+  );
+
+  async function gsGet(clientAuth) {
+    const gsApi = google.sheets({
+      version: 'v4',
+      auth: clientAuth
+    });
+
+    const options = {
+      spreadsheetId: '1JWYHiOightRfEg3qFrDniItyYII5fgXdi99m11mgSH0',
+      ranges: ['Projects!A4:I', 'Events!A4:G']
+    };
+
+    const data = await gsApi.spreadsheets.values.batchGet(options);
+
+    const projects = data.data.valueRanges[0].values;
+    const events = data.data.valueRanges[1].values;
+
+    const sheetsData = {
+      projects,
+      events
+    };
+
+    return sheetsData;
+  }
+
+  client.authorize((err, tokens) => {
```

Comment:
should use promise or better yet...go learn "async await"

async await is the proper way to do async calls....please fix this in next PR

Reply:
will fix in next PR